src/transformers/models/t5/modeling_t5.py

```diff
@@ -340,7 +340,7 @@ def __init__(self, config: T5Config):
     def forward(self, hidden_states):
         forwarded_states = self.layer_norm(hidden_states)
         forwarded_states = self.DenseReluDense(forwarded_states)
-        hidden_states = self.adapter_layer_forward(hidden_states, self.dropout(forwarded_states), None)
+        hidden_states = self.adapter_layer_forward(self.dropout(forwarded_states), hidden_states, None)
```

Comment:
We could make it super obvious which one the residual connection is by using `hidden_states=..., residual_connection=...` in the method call. This could help people adding new models when they have a look at the currently implemented models.